library/swift/BUILD

```diff
@@ -9,45 +9,7 @@ licenses(["notice"])  # Apache 2
 swift_library(
     name = "ios_lib",
     srcs = glob([
-        "DirectResponse.swift",
-        "Engine.swift",
-        "EngineBuilder.swift",
-        "EngineImpl.swift",
-        "EnvoyError.swift",
-        "FinalStreamIntel.swift",
-        "Headers.swift",
-        "HeadersBuilder.swift",
-        "HeadersContainer.swift",
-        "KeyValueStore.swift",
-        "LogLevel.swift",
-        "NetworkMonitoringMode.swift",
-        "PulseClient.swift",
-        "PulseClientImpl.swift",
-        "RequestHeaders.swift",
-        "RequestHeadersBuilder.swift",
-        "RequestMethod.swift",
-        "RequestTrailers.swift",
-        "RequestTrailersBuilder.swift",
-        "ResponseHeaders.swift",
-        "ResponseHeadersBuilder.swift",
-        "ResponseTrailers.swift",
-        "ResponseTrailersBuilder.swift",
-        "RetryPolicy.swift",
-        "RetryPolicyMapper.swift",
-        "RouteMatcher.swift",
-        "Stream.swift",
-        "StreamCallbacks.swift",
-        "StreamClient.swift",
-        "StreamClientImpl.swift",
-        "StreamIntel.swift",
-        "StreamPrototype.swift",
-        "TestEngineBuilder.swift",
-        "Trailers.swift",
-        "UpstreamHttpProtocol.swift",
-        "filters/*.swift",
-        "grpc/*.swift",
-        "mocks/*.swift",
-        "stats/*.swift",
```

Comment:
Personally, I'd prefer we explicitly enumerate source files in build targets and avoid blobs. I feel it obscures information about what's being pulled in, and it has the potential to lead to surprises when files are moved around.

Upstream Envoy tends to follow the pattern of explicit enumeration of source files as well, even when that list is large.

Reply:
Done. I don't feel strongly but the previous approach was inconsistent with a mix of explicit files and globs.

As of 4d77493 everything is being explicitly set.